src/App.js

```diff
@@ -14,11 +14,70 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [...goodsFromServer],
+    selectGoods: [],
+  };
+
+  changeGoods = (good, sel) => {
+    if (!sel.includes(good)) {
+      this.setState({ selectGoods: [...sel, good] });
+    } else {
+      const temp = sel.filter(x => x !== good);
+
+      this.setState({ selectGoods: [...temp] });
+    }
+  };
+
+  clearGoods = () => {
+    this.setState({ selectGoods: [] });
+  };
+
+  render() {
+    const { goods, selectGoods } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          Selected good:
+          <span className="App__title">{selectGoods.join(', ')}</span>
+        </h1>
+        <button
+          type="button"
+          className="App__button"
+          onClick={this.clearGoods}
+        >
+          X
+        </button>
+        <ul className="App__list">
+          {goods.map(good => (
+            <div className="App__wrapper">
+              <li
+                key={good}
+                className={selectGoods.includes(good)
+                  ? 'App__item App__item--selected'
+                  : 'App__item'}
```

Comment:
It would be nice to use [ClassNames](https://www.npmjs.com/package/classnames) library here